R/utils-omnipath.R

```diff
@@ -104,6 +106,7 @@ get_collectri <- function(organism='human', split_complexes=FALSE, ...){
       organism = organism,
       genesymbol=TRUE,
       loops=TRUE,
+      extra_attrs = TRUE,
       ...
     ),
     error = function(e){
```

Comment:
This fallback won't have the `extra_attrs` column (though we can include it in the future). It raises a warning, so the user will be aware. Optionally we can handle the rare case when this happens with `load_meta = TRUE`.